src/main/java/crazypants/enderio/machine/sagmill/TileSagMill.java

```diff
@@ -132,17 +132,6 @@ protected IPoweredTask createTask(IMachineRecipe nextRecipe, float chance) {
   }
 
   @Override
-  public void writeCustomNBT(NBTTagCompound nbtRoot) {
-    super.writeCustomNBT(nbtRoot);
-    lastSendGbScaled = getBallDurationScaled(16);
```

Comment:
The problem here is that this variable is not stored, it is *set* upon writing of NBT. I'd ask @HenryLoenwind how to handle this edge case, since the `@Store` system does not seem able to.

Reply:
I need to look into what that variable actually does... I'd recommend to put the old code into a FIXME comment for now

Reply:
2 way to handle this:

1. drop it and live with one additional PacketGrindingBall after the TE update packet
2. add a onAfterDataPacketSent() callback to the super class for it

I can live with no 1